packages/ez-dev/jest/snapshots/recipes/area/AreaChart.spec.tsx.snap

```diff
@@ -17,9 +17,10 @@ exports[`AreaChart renders an area chart 1`] = `
               <g class="ez-points">
                 <g class="ez-area">
```

Comment:
The unit test should show two areas instead of a single one.

Reply:
This is the area chart